src/common/types/time.cpp

```diff
@@ -112,15 +112,25 @@ static bool TryConvertTime(const char *buf, dtime_t &result) {
 		}
 	}
 
+	if (strict) {
```

Comment:
Shouldn't this be either !strict or always done? It seems like skipping spaces is not exactly strict behavior.

Reply:
Even in strict casting I want to allow for trailing spaces, just no other character. I made this more explicit.